src/gulpfile.js

```diff
@@ -160,7 +160,10 @@ gulp.task('common:build', ['clean'], function() {
     return gulp.start('copy:vendor-css', 'copy:vendor-images', 'sass', 'collectstatic');
 });
 
-gulp.task('test', sequence(['jshint',
+gulp.task('test', sequence([
+            'development',
+            'production',
```

Comment:
I thought perhaps it was copying something that wasn't getting deleted, and was unsure why Jenkins was having issues that weren't appearing locally.  Merged so Jenkins could run it.  This has been reverted in #58, which I tested manually on the Jenkins VM.